sgx-types/src/miscselect.rs

```diff
@@ -0,0 +1,12 @@
+bitflags::bitflags! {
+    /// Section 38.7.2
+    pub struct MiscSelect: u32 {
+        const EXINFO = 1 << 0;
+    }
+}
+
+impl Default for MiscSelect {
+    fn default() -> Self {
+        MiscSelect::EXINFO
```

Comment:
I think we need to default to `MiscSelect::empty()`. We can't be sure that `EXINFO` is setup by whoever calls this.

Reply:
Never mind, the kernel enforces the presence of this flag. Leave it as it is.